src/native/util/common.h

```diff
@@ -34,17 +35,34 @@ namespace noobaa
 
 #define LOG(x) std::cerr << LOG_PREFIX() << x << std::endl
 
+#define SYS_LOG(x)                                      \
+do {                                                    \
+        std::ostringstream oss;                         \
+        oss << "" << LOG_PREFIX() << x << std::endl;    \
+        std::string log_str = oss.str();                \
+        const char* log_msg = log_str.c_str();          \
+        int facility = LOG_LOCAL0;                      \
+        int priority = 0;                               \
+        ::syslog(priority | facility, "%s", log_msg);   \
+    } while (0)
 // to use DBG the module/file should use either DBG_INIT or DBG_INIT_VAR.
 #define DBG_INIT(level) static int __module_debug_var__ = level
 #define DBG_INIT_VAR(debug_var) static int& __module_debug_var__ = debug_var
 #define DBG_SET_LEVEL(level) __module_debug_var__ = level
 #define DBG_GET_LEVEL() (__module_debug_var__)
 #define DBG_VISIBLE(level) (level <= __module_debug_var__)
-#define DBG(level, x)                        \
-    do {                                     \
-        if (DBG_VISIBLE(level)) {            \
-            LOG("[L" << level << "] " << x); \
-        }                                    \
+#define DBG(level, x)                                                   \
+    do {                                                                \
+        if (DBG_VISIBLE(level)) {                                       \
+            const char* syslog_enabled = std::getenv("LOG_TO_SYSLOG_ENABLED"); \
+            const char* stderr_enabled = std::getenv("LOG_TO_STDERR_ENABLED"); \
+            if (syslog_enabled && strcmp(syslog_enabled, "true") == 0) {       \
+                SYS_LOG("[L" << level << "] " << x);                           \
+            }                                                                  \
+            if (stderr_enabled && strcmp(stderr_enabled, "true") == 0) {       \
```

Comment:
I suggest to move this logic inside the LOG macro, and add a separate STD_LOG macro, and then the usage of the LOG and DNG macros will stay the same, like this for example
```cpp
#define STD_LOG(x) std::cerr << LOG_PREFIX() << x << std::endl

#define SYS_LOG(x)                                      \
  ...

#define LOG(x) do { \
            if (syslog_enabled && strcmp(syslog_enabled, "true") == 0) {       \
                SYS_LOG("[L" << level << "] " << x);                           \
            }                                                                  \
            if (stderr_enabled && strcmp(stderr_enabled, "true") == 0) {       \
                STD_LOG("[L" << level << "] " << x);                        \
            } \
    } while (0)

```